src/components/layout/footer.tsx

```
@@ -0,0 +1,74 @@
+        message: string
+        author: {
+            date: string
+        }
+    }
+}
+
+const GITHUB_API_URL = 'https://api.github.com/repos/remcostoeten/ripgrep-command-creator-helper/commits'
+const BASE_VERSION = { major: 1, minor: 0, patch: 1 }
+
+let latestCommitCache: { data: CommitData | null, timestamp: number } = { data: null, timestamp: 0 }
+
```

ESLint

[error] Missing semicolon.

(semi)